### ISSUE TITLE
why don't you try training with ground truth mask?

### ISSUE THREAD

**Issue by kkyi10** [junzhezhang/mesh-inversion#5]
Hello, thank you very much for the excellent work, I have one more question, in fact from the visualization of the paper, your method outperforms the existing methods very well, but why does IOU seem to be similar to sota method in quantitative evaluation, and why don't you try training with ground truth mask? Looking forward to hearing from you:)

**Comment by junzhezhang** [comment]
The quantitative results are in 3D IoU but we use 2D estimated mask during inference; the geometric performance is to some extent bottlenecked by the pre-trained ConvMesh performance - that is why in terms of 3D IoU we do see noticeable out-performance.

We use estimated masks instead of GT masks just for a fair comparison. Just to re-emphasize it is test-time optimziation instead of training.

Hope it clarifies.

Junzhe

**Comment by kkyi10** [comment]
> Thank you for your reply! Yes, the quantitative results are in 3D IoU,your method shows significantly more refined shape reconstruction than SMR, such as beak and foot. However, in terms of the quantitative results, why is the inference similar even if the estimated mask is used.
![1669902658326](https://user-images.githubusercontent.com/65757371/205070451-ab661313-0421-4e14-8eed-ac02d38dbcef.png)
And you mentioned that the estimated mask is used for fair comparison, in fact, the other methods seem to take the true mask as annotation. So I don't quite understand this result, maybe I'm getting it wrong somewhere?

**Comment by junzhezhang** [comment]
@kkyi10 Hi, I would like to clarify that, Table 1 is indeed in 2D IoU  (measured between our output mask and GT mask), whereas Table 3 is in 3D IoU as we do have 3D GT for Pascal3D Car dataset, but 3D GT is not available for CUB dataset.

Estimated mask is used for test-time optimization for fair comparison, but not for computing the 2D IoU(as mentioned above).

**Comment by kkyi10** [comment]
Ok, I understand it. Thank you very much for your reply! ：）
